aptos-move/aptos-vm/src/data_cache.rs

```diff
@@ -98,6 +99,27 @@ impl<'block, S: StateView> StateView for StateViewCache<'block, S> {
         }
     }
 
+    fn get_state_value(&self, state_key: &StateKey) -> anyhow::Result<Option<Vec<u8>>> {
+        //TODO: Add a caching layer on this once the VM write set starts populating state_value changes.
```

Comment:
yeah.. we need it soon-ish

Reply:
Once we have write set API changes for table, we can add this.